src/dmd/astbase.d

```diff
@@ -6810,13 +6810,27 @@ struct ASTBase
     {
         extern (C++) __gshared int ptrsize;
 
+        enum OS : ubyte
+        {
+            linux        = 1,
+            Windows      = 2,
+            OSX          = 4,
+            OpenBSD      = 8,
+            FreeBSD      = 0x10,
+            Solaris      = 0x20,
+            DragonFlyBSD = 0x40,
+            all = linux | Windows | OSX | OpenBSD | FreeBSD | Solaris | DragonFlyBSD,
+            Posix = linux | OSX | OpenBSD | FreeBSD | Solaris | DragonFlyBSD,
+        }
+
+        extern (C++) __gshared OS os;
         extern (C++) static Type va_listType(const ref Loc loc, Scope* sc)
```

Comment:
I think that this function should be here: the parser does not use it and it isn't used internally by ASTBase. You could try deleting it, that way you do not have to duplicate the code.